assets/new-hearing-form.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script type="text/javascript" src="//api.filepicker.io/v2/filepicker.js" charset="utf-8">
+    </script>
+    <link href="../../css/bootstrap.min.css" rel="stylesheet">
+    <link href="../../css/admin-form.css" rel="stylesheet">
+  </head>
+
+  <body>
+    <div class="form-container container-fluid">
+      <form action="/hearings/" method="post">
+        <nav class="header navbar">
+          <h2 id="title">Project Name To Be Updated</h2>
+          <button type="submit" class="navbar-right btn btn-default publish">Submit Hearing</button>
+          <span class="navbar-right h4 email-us">E-mail Us</span>
+        </nav>
+
+        <div class="project navbar">
+          <div class="row">
+            <div class="col-xs-4">
+              <label for="date">Hearing Date</label>
+              <input type="date" name="date" value="">
+            </div>
+            <div class="col-xs-4">
+              <label for="location">Location</label>
+              <input type="text" name="location" value="">
+            </div>
+            <div class="col-xs-4">
+              <label for="start-time">Start Time</label>
+              <input type="time" name="start-time" value="">
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-xs-4">
+              <label for="hearing-type">Hearing Type </label>
+              <select class="" name="hearing-type">
```

Comment:
Oh, there are actually some fields we should add to this form - See @dbarnhart's updated schema in server.js